packages/web-components/package.json

```diff
@@ -58,9 +57,13 @@
     "@types/chai-spies": "^1.0.1",
     "@types/karma": "^5.0.0",
     "@types/mocha": "^7.0.2",
+    "@typescript-eslint/eslint-plugin": "4.22.0",
+    "@typescript-eslint/parser": "4.22.0",
     "chai": "^4.2.0",
     "chai-spies": "1.0.0",
     "circular-dependency-plugin": "^5.0.2",
+    "eslint": "7.25.0",
```

Comment:
For dev deps that are already listed at the root, you probably shouldn't list them in your package.json as well unless the version needs to be different.